openhtmltopdf-core/src/main/java/com/openhtmltopdf/render/BlockBox.java

```diff
@@ -417,7 +420,6 @@ private MarkerData.TextMarker makeTextMarker(LayoutContext c, IdentValue listSty
         if (listDirection == IdentValue.RTL) {
             text = "  .".concat(text);
         } else {
-            assert listDirection == IdentValue.LTR || listDirection == IdentValue.AUTO;
```

Comment:
was this assertion invalidated by your changes?

Reply:
It's a long time ago that past me worked on this! I likely took it out as being redundant: the _getDirection()_ call above only passes valid _CSSName.DIRECTION_ values, which are already restricted to the set in _PrimitivePropertyBuilders.DIRECTIONS_. I have no strong objection to keeping it if you want the additional security.

Reply:
Of course it's not a big deal, but I personally would like to keep it for the benefit of a hypothetical future developer who might read it and understand the code a little better.